bip-0197.mediawiki

```diff
@@ -0,0 +1,156 @@
+<pre>
+  BIP: 197
+  Layer: Applications
+  Title: Hashed Time-Locked Collateral Contract
```

Comment:
Needs to match README

Reply:
Changed title in README for BIP 197 to `Hashed Time-Locked Collateral Contract`